src/js/stores/MesosStateStore.js

```diff
@@ -90,22 +90,35 @@ class MesosStateStore extends GetSetBaseStore {
       .subscribe(this.onStreamData, this.onStreamError);
   }
 
+  getLastMesosState() {
+    return Object.assign(
+      {
+        executors: [],
+        frameworks: [],
+        master_info: [],
```

Comment:
`master_info` is an object http://mesos.apache.org/documentation/latest/operator-http-api/#get_master

Reply:
Thank you for noticing, fixed that 👍